compiler/rustc_codegen_llvm/src/gotoc/utils.rs

```diff
@@ -48,7 +48,12 @@ impl<'tcx> GotocCtx<'tcx> {
             Ok(pathbuf) => pathbuf.to_str().unwrap().to_string(),
             Err(_) => filename0,
         };
-        Location::new(filename1, self.current_fn.as_ref().map(|x| x.name()), line, Some(col))
+        Location::new(
+            filename1,
+            self.current_fn.as_ref().map(|x| x.readable_name().to_string()),
```

Comment:
Do we do this operation anywhere else? If so it might make sense to make this a function